rpcs3/Emu/RSX/Overlays/overlay_fonts.cpp

```diff
@@ -107,19 +108,8 @@ namespace rsx
 			glyph_load_setup result;
 			result.font_names.push_back(font_name);
 
-#ifdef _WIN32
-			result.lookup_font_dirs.emplace_back("C:/Windows/Fonts/");
-#else
-			char* home = getenv("HOME");
-			if (home == nullptr)
-				home = getpwuid(getuid())->pw_dir;
-
-			result.lookup_font_dirs.emplace_back(home);
-			if (home[result.lookup_font_dirs[0].length() - 1] == '/')
-				result.lookup_font_dirs[0] += ".fonts/";
-			else
-				result.lookup_font_dirs[0] += "/.fonts/";
-#endif
+			auto font_dirs = Emu.GetCallbacks().get_font_dirs();
```

Comment:
```suggestion
			const std::vector<std::string> font_dirs = Emu.GetCallbacks().get_font_dirs();
```

Just to explain these nitpicks regarding auto:
I'm a strong believer in using explicit types whenever possible.
If you review or investigate code on a daily basis it is very confusing or at least inconvenient to always have to double check the code in the IDE to see what type you're looking at.
(It's fine for really complex types or critical types where a typo may slip by the compiler like iterators etc.)